en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -1,7 +1,7 @@
 # Change the Default Mediation Flow of API Requests
 
 !!! note
-This tutorial uses the WSO2 API Manager Tooling Plug-in.
+    This tutorial uses the WSO2 API Manager Tooling Plug-in.
```

Comment:
Is there another way to do it?

Reply:
We can provide the mediation policy directly and ask to upload it as a custom mediation policy.